open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -4114,7 +4116,9 @@ END
 
 PROCEDURE CHECKFORINVALIDCHARS
```

Comment:
Didn't see where the value of the group names were being checked before being passed back to the UX. Last I checked, the CONFIG program allowed the use of invalid chars when running the config program and defining a group name.